package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "sequelize-auto",
-  "version": "0.4.29",
+  "name": "@tkay/sequelize-auto",
```

Comment:
We can't accept this PR due to this file being committed

Reply:
@durango fixed. please check